src/services/message.service.ts

```diff
@@ -77,6 +77,36 @@ export class MessageService {
     return msg;
   }
 
+  public generateEmbedList(title: string, listItems: string[]): MessageEmbed {
+    const maxCol = 3;
+    const maxRows = 10; // This is a soft limit
+
+    // Setting up number of columns and rows for the list
+    const numCols = Math.min(maxCol, Math.ceil(listItems.length / maxRows));
+    const numRows = Math.ceil(listItems.length / numCols);
+
+    // Sets up the embed
+    const embedItem = new MessageEmbed();
+    embedItem.setColor('#0099ff').setTitle(title);
```

Comment:
Dont set the title in this function to make it simpler. bc each embed will need a different title, its best to set it in the plugin

Reply:
perhaps color also configurable. maybe make options Object w optional params

Reply:
Easier to let the plugin do it all @c650 when the messageEmbed is returned (I tried to send in a optional messageEmbed but the foreach didn't like it).  So I'll just take out the title like tanndlin said and the plugin can deal with it.  Just thought it was common enough to stay in it.

Reply:
gotcha.

Reply:
In the end, I made it a dynamic choice.